app/models/katello/repository.rb

```diff
@@ -724,6 +725,33 @@ def custom_docker_repo_path
       parts.map { |x| x.gsub(/[^-\w]/, "_") }.join("-").downcase
     end
 
+    def repository_type
+      RepositoryTypeManager.find(self.content_type)
+    end
+
+    def index_linked_repo
+      if (base_repo = self.target_repository)
```

Comment:
Typo?

Reply:
mentioned in person, but this is assigning self.target_repository to the 'base_repo' var and then evaulating if its truthy